lib/usecases/log/GetAllLogsUseCase.js

```diff
@@ -29,6 +29,27 @@ const { ApiConfig } = require('../../config/index.js');
 const { Op } = require('sequelize');
 const { dataSource } = require('../../database/DataSource.js');
 
+/**
+ * Checks if there is an exclusion sign in front of the variable dtoField, and
+ * removes the exclusion sign, if present.
+ *
+ * @param {string} dtoFilter the value to check for an exclusion sign.
+ * @returns {object} represents the parsed result.
+ * @property {boolean} excl indicates whether the first character of dtoFilter
+ *      is equal to exclSign.
+ * @property {string} strippedFilter the filtered string with the first
+ *      character removed if excl is true, otherwise, the original dtoFilter.
+ */
+const checkForFilterExclusion = (dtoFilter) => {
```

Comment:
I would recommend that you add this method as a general method in a separate file so that other services can potentially re-use it